packages/jokul/src/components/table/TableHead.tsx

```diff
@@ -10,9 +10,11 @@ export interface TableHeadProps
 }
 
 const TableHead = forwardRef<HTMLTableSectionElement, TableHeadProps>(
-    ({ className, srOnly, sticky, ...rest }, ref) => {
+    ({ className, srOnly, sticky = false, ...rest }, ref) => {
```

Comment:
Ville det ikke holdt med kun denne endringen (og fjerne `undefined` fra typen til `hasStickyHead`? Er ikke så fan av å innføre effekter for derived state hvis vi kan unngå det.

Reply:
Problemet er ikke så mye default-verdien som at vi oppdaterer Table mens TableHeader rendrer samtidig som den er en child av Table. Effekten gjør at hele treet er mountet før oppdateringen i parent skjer. 

Det fungerte for de som satte verdien til `false` fordi det ikke endte med en oppdatering, men hvis vi også vil bli kvitt feil for de som setter sticky til `true` så vet jeg ikke om noen andre triks.

Reply:
Kanskje det hadde fungert dersom `Table` ikke var sin egen context-provider men at det var en egen komponent, det er jeg litt usikker på.

Reply:
Aah, ja ser den. `TableHead` setter verdi i context, som igjen rerendrer `Table`, som igjen rerendrer `TableHead` 😵‍💫

Reply:
Testet nettopp med en dedikert Context og da flyttet bare feilen seg dit. Vi kaller en parent sin `setState` synkront i render og det gjør baby react sint 😭

Reply:
Ja, så lenge endring i state tvinger rerender av både `Table` og `TableHead`, og den ene er child av den andre, blir det nok krøll…